src/content/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: Triage and run diagnostics in eight clicks
+tags:
+  - APM
+  - Triage
+  - Diagnostics
+metaDescription: Use our application monitoring tools to quickly triage and run diagnostics during an incident.
+---
+import apmCheckoutServiceinApm from 'images/apm_screenshot-full_checkout-service-in-apm.png'
+
+import apmTimePickerinApm from 'images/apm_screenshot-full_time-picker-in-apm.png'
+
+import accountsSelectSlowestTransaction from 'images/accounts_screenshot-crop_select-slowest-transaction.gif'
+
+import apmInvestigateThroughputinAPM from 'images/apm_screenshot-full_investigate-throughput-in-APM.gif'
+
+import apmTransactionDetailsPage from 'images/apm_screenshot-crop_Transaction-details-page.png'
+
+import apmTransactionBreakdown from 'images/apm_screenshot-crop_transaction-breakdown.png'
+
+import apmThroughputTransactionsDetailsPage from 'images/apm_screenshot-crop_throughput-transactions-details-page.gif'
+
+import apmTraceDetails from 'images/apm_screenshot-crop_trace-details.png'
+
+import apmTransactionTracesinAPM from 'images/apm_screenshot-full_transaction-traces-in-APM.gif'
+
+import apmTransactionsTabinApmOverview from 'images/apm_screenshot-full_transactions-tab-in-apm-overview.gif'
+
+
+We all know what it's like to wake up in the morning to a ping from your leadership team telling you that something is going wrong and they need it fixed as soon as possible. This is where our APM tools come in. When you have limited time and increased urgency, you can use APM to run diagnostics and triage issues in eight clicks. 
+
+For this example, we're going to explore a situation where you work for an e-commerce company and you manage the checkout team. You are alerted to the fact that the `Pay Now` button is failing and customers can't checkout. 
```

Comment:
Even for imaginary/external UI coopy, we should try to abide by our sentence case guides. (and, as an aside, if you didn't know, even though we now have title case in UI for many things, we should still use sentence case for those things when referencing them.) 
```suggestion
For this example, we're going to explore a situation where you work for an e-commerce company and you manage the checkout team. You are alerted to the fact that the `Pay now` button is failing and customers can't checkout. 
```

Reply:
should be "can't check out" I thnk